diff -Nur pkgconfig-0.7.0.orig/Makefile.am pkgconfig-0.7.0/Makefile.am
--- pkgconfig-0.7.0.orig/Makefile.am	Wed Jun  6 00:12:59 2001
+++ pkgconfig-0.7.0/Makefile.am	Tue Jun 26 23:42:27 2001
@@ -1,5 +1,3 @@
-SUBDIRS = glib-1.2.8
-
 m4dir = $(datadir)/aclocal
 m4_DATA = pkg.m4
 
@@ -8,23 +6,15 @@
 
 bin_PROGRAMS = pkg-config
 
-INCLUDES=-DPKGLIBDIR="\"$(pkglibdir)\"" -I./glib-1.2.8
+INCLUDES=-DPKGLIBDIR="\"$(pkglibdir)\"" @GLIB_CFLAGS@
 
 pkg_config_SOURCES= \
 	pkg.h \
 	pkg.c \
-	partial-glib.h \
-	partial-glib.c \
 	parse.h \
 	parse.c \
 	main.c \
 	findme.c \
-	findme.h \
-	popt.c \
-	popt.h \
-	poptconfig.c \
-	popthelp.c \
-	poptint.h \
-	poptparse.c
+	findme.h
 
-pkg_config_LDADD=glib-1.2.8/libglib.la
+pkg_config_LDADD=@GLIB_LIBS@
diff -Nur pkgconfig-0.7.0.orig/configure.in pkgconfig-0.7.0/configure.in
--- pkgconfig-0.7.0.orig/configure.in	Wed Jun  6 01:15:29 2001
+++ pkgconfig-0.7.0/configure.in	Tue Jun 26 23:44:18 2001
@@ -1,8 +1,6 @@
 
 AC_INIT(pkg-config.1)
 
-AC_CONFIG_SUBDIRS(glib-1.2.8)
-
 AM_INIT_AUTOMAKE(pkgconfig, 0.7.0)
 
 AM_MAINTAINER_MODE
@@ -14,6 +12,11 @@
 AC_PROG_CC
 
 AC_FUNC_ALLOCA
+
+AC_CHECK_HEADERS(popt.h)
+
+AM_PATH_GLIB(1.2.8,,AC_MSG_ERROR([*** GLIB >= 1.2.8 not installed - please install first ***]))
+AC_CHECK_LIB(popt, main)
 
 AC_CHECK_FUNCS(setresuid setreuid,break)
 
