--- pkgconfig-0.6.0/Makefile.am.wiget	Wed Jun  6 02:54:36 2001
+++ pkgconfig-0.6.0/Makefile.am	Mon Jun 18 17:26:53 2001
@@ -1,5 +1,3 @@
-SUBDIRS = glib-1.2.8
-
 m4dir = $(datadir)/aclocal
 m4_DATA = pkg.m4
 
@@ -8,26 +6,18 @@
 
 bin_PROGRAMS = pkg-config
 
-INCLUDES=-DPKGLIBDIR="\"$(pkglibdir)\"" -I./glib-1.2.8
+INCLUDES=-DPKGLIBDIR="\"$(pkglibdir)\"" @GLIB_CFLAGS@
 
 pkg_config_SOURCES= \
 	pkg.h \
 	pkg.c \
-	partial-glib.h \
-	partial-glib.c \
 	parse.h \
 	parse.c \
 	main.c \
 	findme.c \
-	findme.h \
-	popt.c \
-	popt.h \
-	poptconfig.c \
-	popthelp.c \
-	poptint.h \
-	poptparse.c
+	findme.h 
 
-pkg_config_LDADD=glib-1.2.8/libglib.la
+pkg_config_LDADD=@GLIB_LIBS@
 
 ChangeLog:
 	cvs2cl.pl
--- pkgconfig-0.6.0/configure.in.wiget	Wed Jun  6 02:30:53 2001
+++ pkgconfig-0.6.0/configure.in	Mon Jun 18 17:23:46 2001
@@ -1,7 +1,5 @@
 AC_INIT(pkg.m4.in)
 
-AC_CONFIG_SUBDIRS(glib-1.2.8)
-
 AM_INIT_AUTOMAKE(pkgconfig, 0.6.0)
 ATLEAST="0.6.0"
 AC_SUBST(ATLEAST)
@@ -9,6 +7,11 @@
 AM_MAINTAINER_MODE
 AC_PROG_CC
 AM_PROG_LIBTOOL
+
+AC_CHECK_HEADERS(popt.h)
+
+AM_PATH_GLIB(1.2.8,,AC_MSG_ERROR([*** GLIB >= 1.2.8 not installed - please install first ***]))
+AC_CHECK_LIB(popt, main)
 
 AM_CONFIG_HEADER(config.h)
 AC_FUNC_ALLOCA
